docs/contribution-guide.md

```diff
@@ -1,19 +1,104 @@
+[apidoc]: https://kiltprotocol.github.io/sdk-js/api/index.html
+[jsdoc]: https://www.npmjs.com/package/eslint-plugin-jsdoc
+[typedoc]: https://github.com/TypeStrong/typedoc
+
 # Contribution Guide
 
-## NB
+## Documentation
+
+We want the KILT SDK to be easy to use for all fellow developers. We ❤️ well-documented code.
+
+KILT's API doc is available [here][apidoc].
+
+In the KILT SDK, modules and public methods are documented inline, in the code. These inline "doc blocks" need to follow some linting rules. Make sure ESLint is activated in your code editor, so the errors and warnings are highlighted.
```

Comment:
I think of `inline` as 
```
var bla = 'keks'; // my inline comment
```
not the one on top of a method. Maybe there is a better wording here?